Resources/Prototypes/Loadouts/Jobs/Medical/chief_medical_officer.yml

```diff
@@ -8,6 +8,16 @@
       role: JobChiefMedicalOfficer
       time: 72000 #20 hrs
 
+# For IMP Prestige
+- type: loadoutEffectGroup
+  id: PrestigeCMO
+  effects:
+  - !type:JobRequirementLoadoutEffect
+    requirement:
+      !type:RoleTimeRequirement
+      role: JobChiefMedicalOfficer
+      time: 180000 #50 hrs
```

Comment:
we already have a 50 hour prestige loadoutEffectGroup for cmo: you can use `VeteranCMO` for the loadouts and delete this one since it's essentially a duplicate